common/djangoapps/student/views/management.py

```diff
@@ -172,6 +176,23 @@ def index(request, extra_context=None, user=AnonymousUser()):
     return render_to_response('index.html', context)
 
 
+def show_auto_generated_username(username):
```

Comment:
Can the name be more descriptive? Because we are not showing username based on this check